server/routes/banners/impression/index.ts

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+import doQuery from '../../../model/doQuery'
+import responseHelper from '../../../middlewares/responseHelper';
+
+const router = express.Router();
+
+router.route('/')
+  .get(
+    responseHelper.middleware.withErrorCatch(async (req, res, next) => {
+      const dataString = 'DATE_SUB(NOW(), INTERVAL 60 MONTH)';
+
+      const query = `
+                    SELECT SUM(cL.cashFromMarketer / (4 * mD.unitPrice))
```

Comment:
```suggestion
                    SELECT SUM(cL.cashFromMarketer / (2 * mD.unitPrice))
```
현재 CPM 단가 4원이 아니라 2원이라 2로 변경해야합니다.